modular_skyrat/modules/primitive_structures/code/storage_structures.dm

```diff
@@ -0,0 +1,27 @@
+// Wooden shelves that force items placed on them to be visually placed them
+
+/obj/structure/rack/wooden
+	icon_state = "shelf_wood"
+	icon = 'modular_skyrat/modules/primitive_structures/icons/storage.dmi'
+	resistance_flags = FLAMMABLE
+	flags_1 = NODECONSTRUCT_1
```

Comment:
It'd be fun if they could actually be deconstructed, and be made out of wood by players. It's some nice sprites, after all.

Reply:
true...